pkg/reconciler/workload/placement/placement_reconcile.go

```diff
@@ -81,3 +82,8 @@ func (c *controller) patchPlacement(ctx context.Context, clusterName logicalclus
 	logger.WithValues("patch", string(data)).V(2).Info("patching Placement")
 	return c.kcpClusterClient.Cluster(clusterName).SchedulingV1alpha1().Placements().Patch(ctx, name, pt, data, opts, subresources...)
 }
+
+// listWorkloadAPIBindings list all apibindings in bound phase and the kind is compute
+func (c *controller) listWorkloadAPIBindings(clusterName logicalcluster.Name) ([]*apisv1alpha1.APIBinding, error) {
+	return c.apiBindingLister.Cluster(clusterName).List(labels.Set{workloadv1alpha1.ComputeAPIBindingLabelKey: ""}.AsSelector())
```

Comment:
Can't we simply test on the existence of the label, instead of precisely checking that the value == empty string ?